pkg/disttask/framework/handle/handle.go

```diff
@@ -267,6 +271,21 @@ func GetTargetScope() string {
 	return vardef.ServiceScope.Load()
 }
 
+// GetCloudStorageURI returns the cloud storage URI with cluster ID appended to the path.
+func GetCloudStorageURI(ctx context.Context, store kv.Storage) string {
+	cloudURI := vardef.CloudStorageURI.Load()
+	if s, ok := store.(kv.StorageWithPD); ok {
+		// When setting the cloudURI value by SQL, we already checked the effectiveness, so we don't need to check it again here.
+		u, _ := litstorage.ParseRawURL(cloudURI)
+		if len(u.Path) != 0 {
+			u.Path = path.Join(u.Path, strconv.FormatUint(s.GetPDClient().GetClusterID(ctx), 10))
+			return u.String()
+		}
+	}
+	logutil.BgLogger().Error("Can't get cluster id from store, use default cloud storage uri")
```

Comment:
Can we add a test case to cover this fail to get cluster id path?

Reply:
Currently i can't find a case hit here.
`if s, ok := store.(kv.StorageWithPD); ok {`
But is our code, some places check this type conversion and some did't,  I think it's more safe to do this check, so i add a log here and use the default url if some unexpected occur.